docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -98,13 +98,22 @@ You can take advantage of the [PickersDay](/x/api/date-pickers/pickers-day/) com
 
 You can customize the month layout with some props.
 
+### Show additional days
+
 To shows all days of displayed weeks, included those outside of the current month, use `showDaysOutsideCurrentMonth`.
```

Comment:
Perhaps we can review the props names to be more consistent between each other (in another PR).
For instance, would it make sense to rename this property to `displayDaysOutsideCurrentMonth`?

Reply:
Why not. Can you create an issue for it?